hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/bni/Dispatch.java

```diff
@@ -42,6 +42,14 @@ public interface Dispatch {
     // --- {@code contractCallResult} has the given sender contract number; and whose result is derived
     // --- from the returned state via a given {@link ResultTranslator}
 
+    /**
+     * Returns the all the bytes of entropy available in this scope.
+     *
+     * @return the available entropy
+     */
+    @NonNull
+    Bytes entropy();
```

Comment:
Maybe the type offered up by `Dispatch.entropy()` ought to already be converted to `tuweni.bytes.Bytes` instead of returning the PBJ type?

Reply:
I was trying to leave Besu types out of this interface.

Reply:
I think this interface needs to move to the smart contract module anyway. It is not general functionality and should not be defined in hedera-spi. If we move it there, we can also use classes close to the implementation.